collectors/ebpf.plugin/multi_metadata.yaml

```diff
@@ -28,20 +30,21 @@ modules:
     overview:
       data_collection:
         metrics_description: "Monitor calls for functions responsible to open or close a file descriptor and possible errors."
-        method_description: "Attach tracing (kprobes and trampoline) to internal kernel functions according options used to compile kernel."
+        method_description: "Attach tracing (kprobe and trampoline) to internal kernel functions according options used to compile kernel."
       supported_platforms:
-        include: []
+        include:
+          - Linux
         exclude: []
       multi_instance: true
       additional_permissions:
-        description: "The plugin needs setuid because it loads data inside kernel."
+        description: "The plugin needs setuid because it loads data inside kernel. Netada sets necessary permission during installation time."
```

Comment:
```suggestion
        description: "The plugin needs setuid because it loads data inside kernel. Netdata sets necessary permissions during installation time."
```